### DISCUSSION TITLE
Is there a difference between F-Droid and Google Play Store releases?

### DISCUSSION THREAD

**Discussion by djo5296** [localsend/localsend#358] [Discussion (English, International)] [answered]
title. cuz some F-Droid apps are better, but so far, the ones I have are the same with the Play Store releases so I haven't the need to get F-Droid just to get updates

**Comment by Tienisto** [comment, marked as answer]
There is no difference between both versions.

**Reply by OneOfMany07** [reply]
Currently the Play version is older than F-Droid's (1.13.1 vs. 1.14.0).  Don't know if the approval process is really slow from Google or what, but it's been a month since the build released on here.

Not trying to complain, just bringing it up in case something broke or wasn't submitted.  I've never published anything on either site.  And my search makes it sound like Play should be faster if anything.

Either way, thanks for all your work!

**Reply by Tienisto** [reply]
See #1181. Google still rejected the app due to the `MANAGE_EXTERNAL_STORAGE` permission. I don't have time currently to investigate alternative solutions...
